tests/bwc/test_rolling_upgrade.py

```diff
@@ -225,6 +247,29 @@ def _test_rolling_upgrade(self, path, nodes):
                 # Add the shards of the new partition primaries
                 expected_active_shards += shards
 
+                if int(path.from_version.split('.')[0]) >= 5:
+                    c.execute(f"insert into doc.t2(a, b) values ({idx}, {idx})")
+                    c.execute("refresh table t2")
+                    c.execute(f"select a, b, c>=1 and c<=2, d>a+b from doc.t2 where a = {idx}")
+                    self.assertEqual(c.fetchall(), [[idx, idx, True, True]])
+                    old_version = '.'.join(map(str, node.version))
+                    c.execute("select distinct(version['created']) from information_schema.tables where table_name = 't2'")
+                    self.assertEqual(c.fetchall(), [[old_version]])
+                    if int(path.from_version.split('.')[1]) >= 9:
```

Comment:
After all nodes are upgraded, newly added partitions' versions follow the newer version, starting `5.9`.

Reply:
Please add this info as inline comment